gitlab/tests/test_integration_e2e.py

```diff
@@ -37,6 +38,10 @@ def assert_check(aggregator, metrics):
     for metric in metrics:
         aggregator.assert_metric("gitlab.{}".format(metric))
 
+    aggregator.assert_metrics_using_metadata(
+        get_metadata_metrics(), check_metric_type=False, exclude="gitlab.rack.http_requests_total"
+    )
```

Comment:
Few notes:

1. I think it's better to move `assert_metrics_using_metadata` to `def test_e2e(dd_agent_check)` instead. That way you won't need to set `check_metric_type` to false (integration tests does not report in-app types, but e2e does).

2. `exclude` accepts a list.

Reply:
That makes sense! I'll move the assertion